tuichain/api/views/loans.py

```diff
@@ -340,6 +340,9 @@ def get_loan(request, id):
         )
         loan_dict["funded_value_atto_dai"] = funded_value_atto_dai
 
+        token_contact_adress = controller.loans.get_by_identifier(LoanIdentifier(loan.identifier)).token_contract_adress()
```

Comment:
Should factor out all three occurrences of `controller.loans.get_by_identifier(LoanIdentifier(loan.identifier))` into a variable.

Reply:
Yep, makes sense